Data fetch and display finished

Usagib opened the pull request:
Opening this PR for code review and lintfix purposes.

Hello Team! the features implemented were the following:
- Bootstrap UI created
- UX upgraded
- Open weather API data fetched for current weather
- user view info updated
- live deployed
- esLintFixed

Redvanisation commented on src/index.js:
```diff
@@ -3,10 +3,105 @@ import header from './view/header.js';
+      showWind.innerText = `Wind speed: ${wind.speed} mph`;
+      showHumidity.innerText = `Humidity: ${main.humidity}`;
+
+      if (weather[0].description === 'clear sky') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1506588345361-5e12b7840845?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].main === 'Clouds') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1581788945370-3fe92ade72db?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'broken clouds') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1474218861938-d6b14818c8e2?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1489&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'scattered clouds') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1543226862-39202f29696f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].main === 'Rain') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1428592953211-077101b2021b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'shower rain') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1519692933481-e162a57d6721?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'thunderstorm') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1472145246862-b24cf25c4a36?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1351&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'snow') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1465220183275-1faa863377e3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'mist') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1438803235109-d737bc3129ec?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=666&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+    })
```
[OPTIONAL] You could make a Switch statement for that and put it in a separate module, that would be more concise.

Usagib replied:
Changed if to Switch statement resolved

Redvanisation commented on src/index.js:
```diff
@@ -3,10 +3,105 @@ import header from './view/header.js';
+      }
+    })
+    .catch(() => {
+      const showTemperature = document.getElementById('showTemp');
+      const showWeather = document.getElementById('showWeather');
+      const showMinMax = document.getElementById('showMinMax');
+      const showWind = document.getElementById('showWind');
+      const showHumidity = document.getElementById('showHumid');
+      const showCity = document.getElementById('showCity');
+      showCity.innerText = 'invalid city, you can search by city and country code (Mexico city, MX)';
+      showTemperature.innerHTML = `check country codes <a href="https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes">Here</a>`;
+      showWeather.innerText = '';
+      showHumidity.innerText = '';
+      showWind.innerText = '';
+      showMinMax.innerText = '';
```
[OPTIONAL] You could just display an error message instead.

Usagib replied:
error messsage shown resolved

Usagib commented:
Hello Team!. the following changes had been made according to TSE recommendations:
- changed if statement to switch statement on background change
- changed /src/index.js ln:94-105 to display error message instead

As for the changes required i have some comments / questions:
1. Necessary run common commands like build, development should be in package.json inside `scripts'
- package.json already includes run commands as seen in /package.json ln:6-11, is there any configuration i missed?
2. It includes a rule for the node_modules directory on .gitignore
- .gitignore alredy includes reference to node_modules dir as seen in /.gitignore ln:1, is there any other dir reference i must add?

Thank you for the insight. happy coding.

MauricioRobayo commented:
Great!

Did you make the logo? That's nice!

All previous issues are fixed, you are good to go 🚀 .

Status: __Approved__.

---

[OPTIONAL]: Since you created the [`node_modules`](https://github.com/Usagib/js-weather/tree/03-datafetch/node_modules) directory before including it in `.gitignore` it is on your repo. You might want to remove it, that's why it should be included on `.gitignore` in the first place.

![image](https://user-images.githubusercontent.com/2121481/83896625-11551b80-a71a-11ea-9d52-cb86c2ee7673.png)
